install_config/aggregate_logging.adoc

```diff
@@ -323,9 +323,19 @@ ifdef::openshift-enterprise[]
 endif::openshift-enterprise[]
 |===
 +
-Running the deployer creates a deployer pod and prints its name. Wait until the
-pod is running. This can take up to a few minutes to retrieve the deployer image
-from the registry. You can watch its process with:
+Running the deployer creates a deployer pod and prints its name.
+
+. Wait until the pod is running.
+This can take up to a few minutes to retrieve the deployer image
+from the registry.
++
+In the Kibana interface, you can watch for the aggregation of
```

Comment:
This paragraph is also a little strange. Perhaps the following:

The logs for the default, openshift, and openshift-infra projects are automatically aggregated. These aggregated logs can be found in the ".operations" item in the Kibana interface. These projects are not found individually in the Kibana interface.

The project where you have deployed the EFK stack (eg: logging, as documented here) is **NOT** aggregated into .operations. It will be found under its ID.